internal/pkg/telemetry/telemetry.go

```diff
@@ -0,0 +1,19 @@
+package telemetry
+
+import (
+	"go.opentelemetry.io/otel/codes"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func EndSpan(span trace.Span, errPtr *error) {
```

Comment:
Ukoncenie spanu som dal do samostatnej funkcie, kedze v `opentelemetry` je to viac krokov, a mozno to budeme este ladit. Napr. neviem ci budeme nastavovat `SetStatus -> Err` vsetkym spanom po ceste, alebo iba tomu co je navyssie.